src/clients/SvmFillerClient.ts

```diff
@@ -86,8 +90,32 @@ export class SvmFillerClient {
     this.queuedFills.push({ txPromise: slowFillTxPromise, message, mrkdwn });
   }
 
+  async _executeTxnQueueWithRetry(txPromise: ReadyTransactionPromise, retryAttempt: number): Promise<string> {
+    try {
+      const transaction = await txPromise;
+      const signature = await signAndSendTransaction(this.provider, transaction);
+      const signatureString = signature.toString();
+      return signatureString;
+    } catch (e: any) {
```

Comment:
Minor - `unknown` is preferred over `any` ([link](https://google.github.io/styleguide/tsguide.html#any-unknown))
```suggestion
    } catch (e: unknown) {
```